app/view_models/prior_authority/additional_costs_with_adjustments.rb

```diff
@@ -0,0 +1,132 @@
+module PriorAuthority
+  module RequestedAdditionalCosts
+    def requested_humanized_units
+      if unit_type == 'per_item'
+        "#{requested_items} " \
+          "#{I18n.t('prior_authority.application_details.items.item').pluralize(requested_items)}"
+      else
+        format_period(requested_period)
+      end
+    end
+
+    def requested_humanized_cost_per_unit
+      "#{requested_base_cost_per_unit} " \
+        "#{I18n.t(unit_type, scope: 'prior_authority.application_details.items.per_unit_descriptions')}"
+    end
+
+    def requested_formatted_cost_total
+      NumberTo.pounds(requested_base_cost)
+    end
+
+    private
+
+    def requested_items
```

Comment:
will refactor to remove these unnecessary method, inline with similar changes to servie and travel cost addjustment modules

Reply:
refactored and rebased on top of service and travel cost forms in main